core-java-8/src/main/java/com/baeldung/bifunction/Calculator.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.bifunction;
+
+import java.util.function.BiFunction;
+
+public class Calculator {
+	
+    private Integer a = 0;
+    private Integer b = 0;
+	
+    public Calculator(Integer a, Integer b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    public Integer calculate(Integer a, Integer b, BiFunction<Integer, Integer, Integer> obj){
+        return obj.apply(a, b);
+    }
+	
+    public Integer calculate(BiFunction<Integer, Integer, Integer> obj){

```

Comment:
`obj` is a poor name for the thing that's coming in here. We're asking the calculator to apply a calculation to its own members. Surely the value would called `calculation` or `method` or `operation`